packages/components/src/index.ts

```diff
@@ -0,0 +1,4 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import './styles.css';
+export * from './components';
```

Comment:
```suggestion
export { ... } from './components';
```
use named export please